### PR TITLE
Upgrade Winstone to 5.16 with Jetty 9.4.39.v20210325

### PR DESCRIPTION
**Pull request by olamy**
See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

### Proposed changelog entries

* Upgrade Winstone to 5.16 with Jetty 9.4.39.v20210325
  *  Eclipse Jetty Changelog: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.39.v20210325
  * Winstone changelog: https://github.com/jenkinsci/winstone/releases/tag/winstone-5.16 

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by res0nance — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
I have published the changelog for https://github.com/jenkinsci/winstone/releases/tag/winstone-5.16 . IMHO it is ready to be merged though more reviews would be appreciated.

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

File: war/pom.xml [hunk 1]
```diff
@@ -439,7 +439,7 @@ THE SOFTWARE.
       <plugin>
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>jetty-maven-plugin</artifactId>
-        <version>9.4.38.v20210224</version>
+        <version>9.4.39.v20210325</version>
```
Comment (daniel-beck) [inline war/pom.xml:442]: ❤️

**Review by daniel-beck — APPROVED** [review]
🤷

**Comment by oleg-nenashev** [comment]
Should we expedite it into the tomorrow's weekly release? I am rather +1 for that, better to discover issues early. CC @timja @MarkEWaite 

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by oleg-nenashev** [comment]
There are quite important bugfixes in the changelog.
I suggest to consider backporting of the change to 2.277.x: https://issues.jenkins.io/browse/JENKINS-65280

**Pull request merged**